Make test_replace_quant_view_dequant_with_requantize use GraphBuilder.

eigen-k opened the pull request:
Summary: Use GraphBuilder to create the model for unit testing.

Differential Revision: D74843628

mcremon-meta commented on backends/cadence/aot/tests/test_fusion_ops_passes.py:
```diff
@@ -312,30 +312,28 @@ def forward(self, x):
+        )
+        builder.output(dequant)
+        graph_module = FuseQuantDequantToRequantizePass()(
+            builder.get_graph_module()
```
Nit: same as the other PR, we can use a variable for readability

eigen-k committed:
Make test_replace_quant_view_dequant_with_requantize use GraphBuilder. (#10928)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/10928

Use GraphBuilder to create the model for unit testing.

Reviewed By: zonglinpeng, mcremon-meta

Differential Revision: D74843628
